torch/utils/data/datapipes/__init__.py

```diff
@@ -0,0 +1 @@
+from torch.utils.data.datapipes.iter import *
```

Comment:
Just considering future map datapipe with same name, we should keep the API only within `iter`.